### ISSUE TITLE
Workaround for opacity background around .ReactCrop--crop-selection in IE (11/Edge)

### ISSUE THREAD

**Issue by pedro-couto** [DominicTobias/react-image-crop#87]
Hi guys!

Here is a quick workaround for making the opacity background around the selection box work in IE.
 
Just using box shadow you can make the rest of the element have a dark opacity or whatever color you would like. You can check the following example:

```
.ReactCrop--crop-selection {
  ...
  box-shadow: 0 0 0 9999px rgba(0,0,0,.5);
}
```
Its up to you to make the decision of either make box shadow work just on IE or remove the background.

When using the `box-shadow` I suggest that you wrap the image container with a element which have `overflow: hidden` css rule, to not make the box shadow bleed across all the page.

I have a [codepen](http://codepen.io/pedro-couto/pen/OpypeX) where i just did a box using the box shadow.

Cheers!

**Comment by DominicTobias** [comment]
Hey, that's a good idea! it could simplify the code quite a bit actually. I'll try it out/see what it does to performance etc

**Comment by DominicTobias** [comment]
Thanks for this, it works nicely and simpifies the code as well as makes it compatible with more browsers 👏 I will be releasing it soon

**Comment by DominicTobias** [comment]
Released in 2.0.3
